recipes/cuquantum/meta.yaml

```diff
@@ -0,0 +1,94 @@
+{% set version = "0.1.0" %}
+{% set patch_version = "30" %}
+
+package:
+  name: cuquantum
+  version: {{ version }}.{{ patch_version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/cuquantum/{{ version }}/local_installers/cuquantum-linux-x86_64-{{ version }}.{{ patch_version }}-archive.tar.xz   # [linux64]
+  url: https://developer.download.nvidia.com/compute/cuquantum/{{ version }}/local_installers/cuquantum-linux-sbsa-{{ version }}.{{ patch_version }}-archive.tar.xz     # [aarch64]
+  url: https://developer.download.nvidia.com/compute/cuquantum/{{ version }}/local_installers/cuquantum-linux-ppc64le-{{ version }}.{{ patch_version }}-archive.tar.xz  # [ppc64le]
+
+  sha256: 8ad8e98f14275ffe0de02574be5c86224af1c657c41baf02c16440301ffe0aae  # [linux64]
+  sha256: 3dd04cf08f1323318e0e63a7e28bc904c426ced3367aca809fba7e7beef94063  # [aarch64]
+  sha256: 0eb84eef619a1cfab5870fb585200a9869a197866685252e4ca8187322809554  # [ppc64le]
+
+build:
+  number: 2
+  skip: True  # [win or ppc64le or cuda_compiler_version != "11.2"]
+  script_env:
+    # for some reason /usr/local/cuda is not added to $PATH in ppc64le's docker image
+    - CUDA_HOME  # [ppc64le or aarch64]
+  script:
+    - mkdir -p $PREFIX/include                                            # [linux]
+    - cp -r include/* $PREFIX/include/                                    # [linux]
+    - mkdir -p $PREFIX/lib                                                # [linux]
+    - mv lib/*.so* $PREFIX/lib/                                           # [linux]
+
+    # Patch for conda-forge:
+    # According to the CUDA support matrix, we require glibc 2.27+ on aarch64-sbsa,
+    # while conda-forge is still on 2.17. However, given that powf is the only
+    # offending symbol that needs a newer glibc, it is the easist if we just patch
+    # it.
+    - patchelf --clear-symbol-version powf $PREFIX/lib/libcustatevec.so   # [aarch64]
+  run_exports:
+    - {{ pin_subpackage('cuquantum', max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cuda') }}
+    - sysroot_linux-64 2.17  # [linux]
+  host:
+    - patchelf >=0.12                          # [aarch64]
+    # to suppress ".so not found" errors
+    - cudatoolkit {{ cuda_compiler_version }}  # [linux]
+    - cutensor 1.4.*                           # [linux]
+  run:
+    # Only GCC_3.0 or older symbols present
+    - libgcc-ng >=3.0     # [linux]
+    # Only GLIBCXX_3.4 or older symbols present
+    - libstdcxx-ng >=3.4  # [linux]
+    - cudatoolkit >=11.2,<12
```

Comment:
```suggestion
```

Reply:
I can remove the rest, but the CTK constraint needs to be kept because we have not yet supported all CUDA 11.x.

Reply:
> I can remove the rest, but the CTK constraint needs to be kept because we have not yet supported all CUDA 11.x.

This is done automatically by `compiler('cuda')`.